modules/face/test/test_face_align.cpp

```diff
@@ -38,10 +38,15 @@ TEST(CV_Face_FacemarkKazemi, can_create_default) {
 TEST(CV_Face_FacemarkKazemi, can_loadTrainingData) {
     string filename = cvtest::findDataFile("face/lbpcascade_frontalface_improved.xml", true);
     string configfile_name = cvtest::findDataFile("face/config.xml", true);
+    string modelfilename = "face_landmark_model.dat";
```

Comment:
@saiteja-talluri can you share the file so that it can be merged?

Reply:
@asmorkalov Is there any other way we can merge this. The python bindings are pretty important.

Reply:
@asmorkalov ,@vikasgupta-github
> Where the file is located?

"face_landmark_model.dat" is the name of the output file, the trained kazemi model. 
it will get created, where the script runs

(btw, when building opencv libs, cmake will also download a pretrained version into .cache)

can we merge it now, and add the missing input validation later ?

Reply:
I am not 100% sure, but it looks like an example **face_landmark_model.dat** file is [here](https://github.com/opencv/opencv_3rdparty/tree/contrib_face_alignment_20170818)